### PR TITLE
downgraded "information on" from error to suggestion

### PR DESCRIPTION
**Pull request by deerskindoll**
downgraded "information on" from error to suggestion

based on https://github.com/redhat-documentation/vale-at-red-hat/issues/472

### CODE REVIEW

File: .vale/styles/RedHat/TermsErrors.yml [hunk 1]
```diff
@@ -92,8 +92,7 @@ swap:
   componentization: component-based development|component model|component architecture|shared components
   componentize: develop components
   comprised of: consist of
-  computer farm|computer ranch: server cluster
-  computer farm|computer ranch: server farm
+  computer farm|computer ranch: server cluster|server farm
```
Comment (aireilly) [inline .vale/styles/RedHat/TermsErrors.yml:95]: Why is this change here?

Comment (deerskindoll) [reply .vale/styles/RedHat/TermsErrors.yml:95]: apologies, I wasn't able to commit the change otherwise because of the duplicate.

Comment (aireilly) [reply .vale/styles/RedHat/TermsErrors.yml:95]: Didn't this get fixed in another PR though? You should be able to rebase to pick up the change on this branch so we don't see this change in the PR.

Comment (aireilly) [reply .vale/styles/RedHat/TermsErrors.yml:95]: Nevermind. Let's merge

**Review by aireilly — APPROVED** [review]
